domains/ItzRishabTools.is-cool.dev.json

```diff
@@ -0,0 +1,22 @@
+{
+    "description": "Project Description",
+
+    "domain": "is-cool.dev",
+    "subdomain": "ItzRishabTools",
+
+    "owner": {
+        "repo": "https://github.com/RISHAB-CREATOR/ItzRishabTools/",
+        "email": "itzrishabboss@gmail.com"
+    },
+
+    "record": {
+        "A": ["185.199.108.153", "185.199.109.153"],
+        "AAAA": ["2606:50c0:8000::153", "2606:50c0:8001::153"],
+        "CNAME": "rishab-creator.github.io/ItzRishabTools/",
+        "MX": ["rishab-creator.github.io", "rishab-creator.github.io"],
+        "NS": ["rishab-creator.github.io", "rishab-creator.github.io"],
```

Comment:
Invalid NS records. Note NS records cannot be used in conjunction with any other records.